charts/kubearchive/templates/api_server.yaml

```diff
@@ -14,7 +13,7 @@ spec:
     metadata:
       labels: *labels
     spec:
-      serviceAccountName: {{ .Values.apiServer.serviceAccount }}
+      serviceAccountName: {{ .Values.kubearchive.serviceAccount }}
```

Comment:
I'm pretty sure the name of the service account is .Values.apiServer.name
https://github.com/kubearchive/kubearchive/blob/main/charts/kubearchive/templates/service_account.yaml#L20

Reply:
@maruiz93, can you confirm that this fix is correct?

Reply:
Ah, @skoved, I see what you are saying. I'll wait for @maruiz93 to confirm which is correct.

Reply:
@skoved, I think you are right. I'll change this!